api/tests/routes/backoffice/update_request_test.py

```diff
@@ -130,19 +146,274 @@ def test_list_account_update_requests(self, authenticated_client):
+            == f"Jean-Pierre Impair né(e) le {phone_number_request.birthDate.strftime('%d/%m/%Y')} ({phone_number_request.applicant_age} ans) impair@example.com"
         )
         assert (
             rows[3]["Jeune"]
-            == f"Octave César ({first_name_update_request.user.id}) né le {first_name_update_request.user.birth_date.strftime('%d/%m/%Y')} ({last_name_and_email_update_request.user.age} ans) heritier@example.com"
+            == f"Jean-Pierre Impair ({phone_number_request.user.id}) né(e) le {phone_number_request.user.birth_date.strftime('%d/%m/%Y')} ({phone_number_request.user.age} ans) impair@example.com"
         )
-        assert rows[3]["Modification"] == "Prénom : Octave => Auguste"
+        assert rows[3]["Modification"] == "Téléphone : +33222222222 => +33111111111"
         assert rows[3]["Instructeur"] == "Instructeur du Backoffice"
+
+    def test_list_filter_by_email(self, authenticated_client):
+        specific_submitter_email_request = users_factories.UserAccountUpdateRequestFactory(
+            user__email="notimportant@example.com",
+            email="that.specific.email@example.com",
+            newEmail=None,
+            newFirstName="notimportant",
+        )
+        specific_new_email_request = users_factories.UserAccountUpdateRequestFactory(
+            newEmail="that.specific.email@example.com"
+        )
+        specific_user_email_request = users_factories.UserAccountUpdateRequestFactory(
+            user__email="that.specific.email@example.com", newEmail=None, newFirstName="notimportant"
+        )
+        users_factories.UserAccountUpdateRequestFactory()
+
+        with assert_num_queries(self.expected_num_queries):
+            response = authenticated_client.get(url_for(self.endpoint, q="that.specific.email@example.com"))
+            assert response.status_code == 200
+
+        rows = html_parser.extract_table_rows(response.data)
+        assert len(rows) == 3
+
+        assert {row["Dossier"] for row in rows} == {
+            str(request.dsApplicationId)
+            for request in (
+                specific_submitter_email_request,
+                specific_new_email_request,
+                specific_user_email_request,
+            )
+        }
+
+    @pytest.mark.parametrize(
+        "q, expected_count, expected_application_ids",
+        (
+            ("+33111111111", 2, {"100001", "100002"}),
+            ("0111111111", 2, {"100001", "100002"}),
+        ),
+    )
+    def test_list_filter_by_phone_number(self, authenticated_client, q, expected_count, expected_application_ids):
+        users_factories.UserAccountUpdateRequestFactory(
+            dsApplicationId=100001, newEmail=None, newPhoneNumber="+33111111111"
+        )
+        users_factories.UserAccountUpdateRequestFactory(
+            dsApplicationId=100002, user__phoneNumber="+33111111111", newEmail=None, newPhoneNumber="0111111111"
```

Comment:
On aura forcément le numéro de téléphone normalisé, dans la base de données.
C'est de plus à cette forme normalisée que l'on comparera la requête.
```suggestion
            dsApplicationId=100002, user__phoneNumber="+33111111111", newEmail=None, newPhoneNumber="+33111111111"
```